server/models/StageModel.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose')
+
+const stageSchema = new mongoose.Schema({
+  _id: {
+    type: String,
+    required: true,
+  },
+  dayId: {
+    type: String,
+    required: true,
+  },
+  tripId: {
+    type: String,
+    required: true,
+  },
+  dayIndex: {
+    type: Number,
+    required: true,
+  },
+  stageIndex: {
+    type: Number,
+    required: true,
+  },
+  stagesLongitude: Number,
+  stagesLatitude: Number,
+  destination: String,
```

Comment:
this is currently referred to as "locationName" throughout the app. should we also rename those occurrences to "destination" in this PR

Reply:
actually should we keep it as locationName to distinguish `stage.destination` from `trip.destination`?

Reply:
Sure

Reply:
Fixed! :D